test/Microsoft.ML.Functional.Tests/FunctionalTestBaseClass.cs

```diff
@@ -50,15 +50,12 @@ public FunctionalTestBaseClass(ITestOutputHelper output)
             FullTestName = test.TestCase.TestMethod.TestClass.Class.Name + "." + test.TestCase.TestMethod.Method.Name;
             TestName = test.TestCase.TestMethod.Method.Name;
 
-            // write to the console when a test starts and stops so we can identify any test hangs/deadlocks in CI
-            Console.WriteLine($"Starting test: {FullTestName}");
```

Comment:
❔ Are these getting logged anywhere else?

Reply:
Yes, logged in MLNETTestCase class, not all test class inherinet from this base test class but we can change all tests to use MLNETFact to logging start and end of test

---
In reply to: [354511751](https://github.com/dotnet/machinelearning/pull/4510#discussion_r354511751) [](ancestors = 354511751)